cves/kernel/CVE-2017-17806.yml

```diff
@@ -364,8 +392,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    In the fix commit, the added code includes a an if statement that explicitly checks if the hash algorithm is keyed or not.
```

Comment:
The reader might not know what you meant by whether or not an algorithm is keyed.